conf/brs-default.properties

```diff
@@ -211,7 +211,7 @@ API.SSL_keyStorePassword = password
 # DB.Password=
 
 # We are using file based as default
-DB.Url=jdbc:h2:file:./burst_db/burst;DB_CLOSE_ON_EXIT=FALSE
+DB.Url=jdbc:sqlite:./burst_db/burst.sqlite
```

Comment:
I think this needs to be experimental for a while, better make it the default only after some heavy testing. Please leave both h2 and sqlite with the sqlite version commented out.